src/ant/src/AntennaChecker.cc

```diff
@@ -1049,14 +1049,15 @@ int AntennaChecker::checkAntennas(odb::dbNet* net,
     report_file.open(report_file_name_, std::ofstream::out);
   }
 
-  bool grt_routes = global_route_source_->haveRoutes();
   bool drt_routes = haveRoutedNets();
+  bool grt_routes = false;
+  if (!drt_routes) {
+    logger_->warn(ANT, 15, "No detailed routing found for nets.");
```

Comment:
I don't like having this warning in every run of repair_antennas. And I don't think you need it either, since if you don't have droute or groute, it will fail anyway. You can just remove this warning.